tests/rules/taint_object_funcdef.js

```diff
@@ -0,0 +1,20 @@
+app.get('/ok', async (req, res) => {
+  const tainted = req.params;
+  async.parallel({
+    somefunc(cb) {
+      // ruleid: test
+      return `SELECT * FROM ${tainted}`
+    }
+  })
+})
+
+// this seems to work
+app.get('/ok', async (req, res) => {
+  const tainted = req.params;
```

Comment:
I guess some of these must be parsed as object literals, but it's not as obvious as in https://semgrep.dev/playground/s/17Qx. 🤔